### ISSUE TITLE
AMP page renders multiple times per single request

### ISSUE THREAD

**Issue by iliran11** [vercel/next.js#18745]
# Bug report

## Describe the bug

AMP page renders 6 times per a single request.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Install this [example repository](https://github.com/vercel/next.js/tree/canary/examples/amp):
2. Add `console.log('hello')` statement just before the return statement (line 12) in `/pages/index.js`
3. run the project -`yarn build` && `yarn start`
4. Go to localhost:3000 and watch `hello` printed 6 times in the terminal - Means server executed the render function 6 times) 
## Expected behavior
Render function should execute only once. It happens in non-AMP.

## System information

- OS: macOS Mojave
- Version of Next.js:  Tested on 9.5.2 and 10.0.0
- Version of Node.js: 12.19.0

**Comment by iliran11** [comment]
Hi @timneutkens ,
This bug should be handled by the team?

**Comment by timneutkens** [comment]
Not sure if this is expected or not (did not verify) but put it into the next iteration (sprint) to be investigated.

**Comment by Linux249** [comment]
I see the same in a 'only amp' project - console.log's inside `getServerSideProps` are printed 6 time and 1 or 2 after the page is allready loaded.

**Comment by ijjk** [comment]
Hi, this appears to be a duplicate of https://github.com/vercel/next.js/issues/9212 I'm going to close this since it doesn't appear to be a bug in Next.js and is related to browser extensions.
